### DISCUSSION TITLE
How to analyze a subset of variables?

### DISCUSSION THREAD

**Discussion by khanetor** [jakobrunge/tigramite#369] [Q&A] [answered]
I have a collection of 3000 time-series, each with 2000 data points. Now, I am only interested in the parents of 5 of these 3000 time-series.

Is there a way to run the analysis for these 5 time-series only? I checked the source code, and there is this `selected_variables`, but it is not available in the PCMCI constructor.

**Comment by sagarsimha** [comment, marked as answer]
Hi. 'selected_variables' is now deprecated. You can use the parameter 'link_assumptions' instead. Its usage is described in Section 3 - "Integrating expert assumptions about links" in the tutorial, 
https://github.com/jakobrunge/tigramite/blob/master/tutorials/causal_discovery/tigramite_tutorial_causal_discovery_overview.ipynb

**Reply by khanetor** [reply]
Thanks @sagarsimha ,
Does that mean that, if I have 4 time-series, [0, 1, 2, 3], and tau_max=2, and I am only interested in the parents of 2, my assumption would be
```python
link_assumptions = { 2: 
    { 
        (0,0): "o?o", (0,-1): "o?o", (0,-2): "o?o",
        (1,0): "o?o", (1,-1): "o?o", (1,-2): "o?o",
        (2,-1): "o?o", (2,-2): "o?o", # series 2 conditioned on the lag of itself
        (3,0): "o?o", (3,-1): "o?o", (3,-2): "o?o",
    },
}
```

Is this the idea?
